lib/jnpr/junos/utils/ftp.py

```diff
@@ -97,13 +97,20 @@ def get(self, remote_file, local_path=os.getcwd()):
                 local_file = local_path
         else:
             local_file = local_path
-        try:
-            self.retrbinary('RETR ' + remote_file,
-                            open(local_file, 'wb').write)
-        except Exception as ex:
-            logger.error(ex)
-            return False
-        return True
+
+        with open(local_file, 'wb') as local_fh:
+            def callback(data):
+                local_fh.write(data)
+                args_callback = self._ftpargs.get('callback')
```

Comment:
this line should be moved above def callback, unnecessarily we will call it with each chunk of data received.